init.org

```diff
@@ -242,15 +242,22 @@ Turn on ~use-package~ integration:
 
 * Theme
 
-Load my theme.
+Load my theme without prompting for safety.
+
+#+begin_src emacs-lisp
+  (defun brew-theme ()
+    (interactive)
+    (load-theme 'brew t))
```

Comment:
The optional second arg tells `load-theme` to treat this as safe, and not prompt.